lib/fastlane/plugin_manager.rb

```diff
@@ -0,0 +1,134 @@
+# encoding: utf-8
+
+# Originally from https://github.com/CocoaPods/CLAide/blob/master/lib/claide/command/plugin_manager.rb
```

Comment:
this ought to include CLAide's license as well, since this is a substantial portion of it